apps/src/blockTooltips/DropletBlockTooltipManager.js

```diff
@@ -29,23 +29,43 @@ var DEFAULT_TOOLTIP_CONFIG = {
 };
 
 /**
+ * Simple helper function that will swallow exceptions, and log them as
+ * console.error. This is done because the way that some of our callbacks are
+ * called by droplet, exceptions would bubble down to the droplet code, and
+ * prevent desired behavior (i.e. we fail to transition back to block mode).
+ */
+function swallowErrors(fn) {
+  return function () {
+    try {
+      fn();
+    } catch (err) {
+      if (typeof(console) !== "undefined" && console.error) {
+        console.error(err);
+      }
+    }
```

Comment:
looks like `console.error` is supported in all browsers we support, so, probably ok not to re-throw here (i.e. looks good how you have it)